LocalSettings.php

```diff
@@ -3598,6 +3598,20 @@
 		'default' => true,
 		'trexwiki' => false,
 	),
+	
+	// Moderation settings
+	'wgModerationNotificationEnable' => array( 
+		'default' => false, // Disable notifications
+		'scruffywiki' => true,
+		'sdiywiki' => true,
+	),
+	'wgModerationNotificationNewOnly' => array(
+		'default' => false, // Notify administrator only about new pages requests
+	),
+	'wgModerationEmail' => array(
+		'default' => $wgEmergencyContact, // Email address to send notifications
+		'sdiywiki' => admin@sdiy.info,
```

Comment:
I believe this needs to be a string, eg `'admin@sdiy.info'`

Reply:
thanks